fuzz/fuzz_targets/streams.rs

```diff
@@ -0,0 +1,66 @@
+#![no_main]
+
+use arbitrary::{Arbitrary, Unstructured};
+use libfuzzer_sys::fuzz_target;
+
+extern crate proto;
+use proto::fuzzing::{ResetStream, Streams, TransportParameters};
+use proto::{Dir, Side, StreamId, VarInt};
+
+#[derive(Arbitrary, Debug)]
+struct StreamParams {
+    side: Side,
+    max_remote_uni: u16,
+    max_remote_bi: u16,
+    send_window: u16,
+    receive_window: u16,
+    stream_receive_window: u16,
+    dir: Dir,
+    transport_params: TransportParameters
+}
+
+#[derive(Arbitrary, Debug)]
+enum Operation
+{
```

Comment:
Non-blocking style nit:
```suggestion
enum Operation {
```